### PR TITLE
Misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -I ../dealii-word-whitelist.txt -S ./bundled`

### CODE REVIEW

File: source/lac/cuda_sparse_matrix.cu [hunk 1]
```diff
@@ -545,7 +545,7 @@ namespace CUDAWrappers
                                  row_ptr_dev.get(),
                                  row_sums.get_values());
 
-#  ifdef DEUG
+#  ifdef DEBUG
```
Comment (drwells) [inline source/lac/cuda_sparse_matrix.cu]: Nice catch!

File: tests/grid/grid_in_ucd_01.cc [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 // read in a Triangulation<2,3> from a mesh file writtent in the UCD format
 
-// the meshe used here were generated by Andrea Mola
+// the meshes used here were generated by Andrea Mola
```
Comment (drwells) [inline tests/grid/grid_in_ucd_01.cc]: we only use one mesh in this example: it should be singular.

Comment (luzpaz) [reply tests/grid/grid_in_ucd_01.cc]: Fixed in b141827

**Comment by drwells** [comment]
This looks good to me. Since we did change more than documentation lets be paranoid and rerun the tester.

/rebuild

**Comment by bangerth** [comment]
Very cool -- it's great to say people take on these kinds of tasks as well! Much appreciated, @luzpaz !

**Comment by luzpaz** [comment]
JFYI, if you want to run `codespell -q 3 -I ../dealii-word-whitelist.txt -S ./bundled` yourself here is the whitelist I made:  
```
als
ans
buss
commend
ith
elemente
everytime
mapp
methode
nd
numer
perverse
pres
reenable
strat
substract
superceded
ue
uint
```

**Comment by masterleinad** [comment]
We have `Run codespell` in our release task list so we would do at least before every release.

**Pull request merged**